.github/workflows/android.yml

```diff
@@ -0,0 +1,112 @@
+name: android
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  push:
+    branches:
+      - master
+
+jobs:
+  ci:
+    if: github.event.pull_request.draft == false
+
+    strategy:
+      matrix:
+        arch: [arm64-v8a, x86_64]
+
+    runs-on: ubuntu-latest
+    container: ghcr.io/f3d-app/f3d-ci
+
+    env:
+      VTK_SHA: 35ff12bbc080528fb0dc3f43ac3288e10ce22c85
+      NDK_VERSION: r21e
+      API_LEVEL: 27
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        path: 'source'
+        fetch-depth: 0
+        lfs: 'false'
+
+    - name: Cache NDK
+      id: cache-ndk
+      uses: actions/cache@v2
+      with:
+        path: android-ndk-${{env.NDK_VERSION}}
+        key: android-ndk-${{env.NDK_VERSION}}-2
+
+    - name: Install NDK
```

Comment:
use an action for consistency

Reply:
what do you mean?

Reply:
all dependency in our CI is handled with custom actions in the actions directory, it would be nice to do the same for NDK